src/components/meta-description.js

```diff
@@ -4,6 +4,6 @@ import { getNestedText } from '../utils/get-nested-text';
 
 export default ({ nodeData: { children } }) => (
     <Helmet>
-        <meta property="og:description" content={getNestedText(children)} />
+        <meta name="description" content={getNestedText(children)} />
```

Comment:
I was going to remove this component all together, but realized it could be used elsewhere since it is in the `ComponentFactory`. With that said I cleaned this up to be the meta description, not the og description.